src/lib/storage/vector_compression/fixed_width_integer/fixed_width_integer_utils.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "storage/vector_compression/compressed_vector_type.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+inline bool is_fixed_width_integer(CompressedVectorType type) {
```

Comment:
We use `inline` very rarely. What is you intention here to use it?

Reply:
I only renamed the file (git didn't get that) and I also don't know why it's inlined. I'll just remove the `inline` keyword.

Reply:
without it, linking fails with duplicate symbols:

> duplicate symbol 'opossum::is_fixed_width_integer(opossum::CompressedVectorType)' in:
    src/lib/CMakeFiles/hyrise_impl.dir/import_export/binary/binary_writer.cpp.o
    src/lib/CMakeFiles/hyrise_impl.dir/storage/index/group_key/composite_group_key_index.cpp.o
duplicate symbol 'opossum::byte_width_for_fixed_width_integer_type(opossum::CompressedVectorType)' in:
    src/lib/CMakeFiles/hyrise_impl.dir/import_export/binary/binary_writer.cpp.o
    src/lib/CMakeFiles/hyrise_impl.dir/storage/index/group_key/composite_group_key_index.cpp.o
ld: 2 duplicate symbols for architecture arm64
clang-12: error: linker command failed with exit code 1 (use -v to see invocation)
[254/447] Building CXX object src/test/CMakeFiles/hyriseTest.dir/lib/expression/expression_evaluator_to_values_test.cpp.o

so unless you have a better suggestion i would keep it like that

Reply:
Ah ok, I see. Well, I assume we have discussed this before and settled with this being ok.